packages/app/src/messages/services/message.service.ts

```diff
@@ -149,6 +149,7 @@ export class MessageService {
 
     const isAnyErrorResponseFound = successResponses.length !== messages.length
 
+    // TODO [mw] kurwa gowno!
```

Comment:
![image](https://user-images.githubusercontent.com/17147149/211339368-11bc0cc2-7d01-4a98-8b53-3f9021650c77.png)

Reply:
😅

Reply:
Removed in https://github.com/mudita/mudita-center/pull/1078/commits/5d1aa7079b4c1c5da6ad60388501daeb034f7b20